### ISSUE TITLE
Feature Request: Option (or replace speedtest with) Librespeed

### ISSUE THREAD

**Issue by jgwehr** [arevindh/pihole-speedtest#92]
Like a few others, (#85 and #88) I've been having a LOT of trouble getting the speedtest-cli to install properly.

Rather than investing more time into this and in the interest of avoiding the bandwidth shaping Ookla is (probably) doing, would it make sense to offer alternatives such as https://github.com/librespeed/speedtest ?

**Comment by arevindh** [comment]
Its simple check here https://github.com/arevindh/pihole-speedtest/wiki/Installing-Speedtest-Mod#install-speedtest-cli

**Comment by jgwehr** [comment]
@arevindh Yes, thanks! I meant for this to be more about Librespeed (or any other alternatives). But, in my specific case, those instructions routinely resulted in this error whenever I tried to run the speedtest:

https://unix.stackexchange.com/questions/644442/speedtest-cli-valueerror-invalid-literal-for-int-with-base-10

I've had this mod for a long time (again, thanks!) so I'm open to thinking there might have been some old pieces causing issues. I followed your instructions a few times without any luck. If more details are helpful, let me know.
